MDANSE/Src/MDANSE/Framework/Jobs/DynamicIncoherentStructureFactor.py

```diff
@@ -230,7 +230,7 @@ def run_step(self, index):
 
             rho = np.exp(1j * np.dot(series, qVectors))
             res = correlate(rho, rho[:n_configs], mode="valid").T[0] / (
-                n_configs * qVectors.shape[1]
+                n_configs * rho.shape[1]
```

Comment:
`qVectors.shape[1]` and `rho.shape[1]` should be the same.

Reply:
I have checked it again. You are right, this change makes no difference.